### PR TITLE
Update xfce docs

### PR DESCRIPTION
**Pull request by worldofpeace**
###### Motivation for this change
The module has basically been rewrote and things aren't like they were.
The one bit of these docs that I think might be misleading/pointless is the troubleshooting section.
I'm not sure any of that is valid/relevant anymore.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: nixos/doc/manual/configuration/xfce.xml [hunk 1]
```diff
@@ -30,16 +30,6 @@
   (system wide), put them into your
   <xref linkend="opt-environment.systemPackages"/>.
  </para>
- <simplesect xml:id="sec-xfce-thunar-volumes">
-  <title>Thunar Volume Support</title>
```
Comment (jtojnar) [inline nixos/doc/manual/configuration/xfce.xml]: Maybe this was meant as a proxy for GVFS, when Thunar is used outside of Xfce?

Comment (jtojnar) [reply nixos/doc/manual/configuration/xfce.xml]: But not sure if we need that.

**Review by jtojnar — APPROVED** [review]
Looks good.

File: nixos/doc/manual/configuration/xfce.xml (hunk 1 above)
Comment (worldofpeace) [reply nixos/doc/manual/configuration/xfce.xml]: It doesn't even seem to make sense to me at all either, it says to just enable xfce.

**Pull request merged**